app/Components/InventoryOverview/index.js

```diff
@@ -1,12 +1,13 @@
-import React, { useContext, useEffect, useState, useRef, useReducer } from 'react';
-import { View, StyleSheet, ScrollView, FlatList, Modal } from 'react-native';
+import React, { useContext, useEffect, useState, useRef } from 'react';
+import { View, StyleSheet, ScrollView, FlatList, Modal, PermissionsAndroid, ImageBackground, Text } from 'react-native';
```

Comment:
in iOS this results into a warning when you press the "Export GeoJson" button:

[Tue Jun 02 2020 19:26:40.906]  WARN     "PermissionsAndroid" module works only for Android platform.
[Tue Jun 02 2020 19:26:40.907]  WARN     [ReferenceError: Can't find variable: Alert]

Reply:
Fixed.